tests/test_edgeql_expr_aliases.py

```diff
@@ -1304,6 +1304,446 @@ async def test_edgeql_aliases_schema_types_03(self):
             []
         )
 
+    async def _check_ddl_type_changes(self, ddls_and_expected_types):
```

Comment:
Let's put these tests in test_edgeql_ddl, or somewhere else.

We need to move away from adding lots of DDL-doing tests to QueryTestCase test suites, since it prevents them from parallelizing.